src/script/lua_api/l_object.cpp

```diff
@@ -905,6 +905,21 @@ int ObjectRef::l_get_player_name(lua_State *L)
 	return 1;
 }
 
+// get_player_velocity(self)
+int ObjectRef::l_get_player_velocity(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	Player *player = getplayer(ref);
+	if(player == NULL){
```

Comment:
Its ok if you don't fix them. You only have to fix code lines you actually change.